python/Ganga/GPIDev/Credentials/ICredential.py

```diff
@@ -524,3 +524,4 @@ def timeleftInHMS(self, force_check=False):
         """
         logger.warning("Dummy method used - no information returned")
         return ""
+
```

Comment:
Both this file and `python/Ganga/GPIDev/Credentials/__init__.py ` seem to still have the double new line at the end of the file.